packages/client/src/store/CallState.ts

```diff
@@ -60,6 +60,11 @@ export enum CallingState {
    */
   RECONNECTING = 'reconnecting',
 
+  /**
+   * The call is in the process of migrating from one node to another.
+   */
+  MIGRATING = 'migrating',
+
```

Comment:
`CallState` is a public API that integrators can use to for example display loading indicators. I think they shouldn't care about if we are in `MIGRATING` or `RECONNECTING` state, I'd rather not leak this information to the public API.